dashboard/app/controllers/maker_controller.rb

```diff
@@ -1,7 +1,17 @@
 class MakerController < ApplicationController
   authorize_resource class: :maker_discount, except: :setup
 
+  # Maker Toolkit is currently used in CSD unit 6, which script id 120.
+  # Retrieves the current CSD unit 6 level that the user is working on.
   def home
+    csd_unit_6_script = Script.find(120)
```

Comment:
Is this a reliable way to get information about CSD Unit 6? I'm worried about scenarios where students are in experiments or an alternate version of CSD Unit 6 for some reason. 

Script 118 is "csd6-dlp" and 119 is "csd6-draft" - are these just old versions we keep around for posterity?

Reply:
I think we're just now making the transition from `csd6-draft` to `csd6`. @mrjoshida might know?

---

For referring to the CSD6 script we should use centralized constants instead of a hard-coded ID.  For CSD6 that's defined here:

https://github.com/code-dot-org/code-dot-org/blob/7516f862531462e11d74c80ac69b8d911be8b6ec/lib/cdo/script_constants.rb#L92

And you refer to it like this:

https://github.com/code-dot-org/code-dot-org/blob/f360231b75f98f500ee9e87beaa6ab5eabae943a/dashboard/app/models/script.rb#L461-L468

Just make sure you require the script constants file:

https://github.com/code-dot-org/code-dot-org/blob/f360231b75f98f500ee9e87beaa6ab5eabae943a/dashboard/app/models/script.rb#L21

Finally, you should be able to look up the script by name:

```rb
csd_unit_6_script = Script.find_by_name(Script::CSD6_NAME)
```

---

All that said, I think your concern around students in an experiment or alternate version of the script is valid.  Is the spec specific that we should only ever show CSD6 info here?

Reply:
Yes, the [spec](https://docs.google.com/document/d/1qYETgpZuYiubKVsRprgGzlj6kEb9W7Xc1Gt6as8v3sA/edit#l) says that we should always have info specific for CSD unit 6; I imagine this would only change if Maker Toolkit is later used in a different unit.  I'll update to find the script by name as suggested and confirm with Josh and Poorva.

Reply:
I think it's reasonable to expect that we may have more than one maker script. I don't know that Poorva has fully specced out versioning yet, but that's at least one case in the immediate future where we might have multiple scripts that rely on maker. I'd prefer that we have a flag on the script so that levelbuilders are in control of which scripts expect the maker app.